src/main/java/no/sikt/nva/DublinCoreParser.java

```diff
@@ -97,12 +98,18 @@ private static String extractTitle(DublinCore dublinCore) {
                    .findAny().orElse(new DcValue()).getValue();
```

Comment:
Generell kommentar:
Nå blir det logget at isbn og issn ikke har blitt scraped.

Jeg var litt usikker på om metoden jeg brukte var teit eller ikke. 
Finner du fort ut hvordan du sørger for at felter med isbn og issn ikke blir logget som uscraped?

Reply:
Har ordnet, men nå ser jeg at INVALID_TYPE Problem blir overskrevet av loggen når den ikke er gyldig.